servant-client/src/Servant/Client/Internal/HttpClient.hs

```diff
@@ -65,7 +65,7 @@ import           GHC.Generics
 import           Network.HTTP.Media
                  (renderHeader)
 import           Network.HTTP.Types
-                 (hContentType, renderQuery, statusCode, Status)
+                 (Status, hContentType, renderQuery, statusCode)
```

Comment:
Same here.

Reply:
`stylish-haskell` (that was enabled in my configuration) put constructors before the function.
I could disable it and undo changes in imports.

Reply:
Well, I think it makes sense to undo all changes to files where there are no significant changes now. Formatting files you touch for other reasons is fine.

Reply:
Fixed.